libs/blocks/utils/utils.js

```diff
@@ -26,4 +28,12 @@ const fetchWithTimeout = async (resource, options = {}) => {
   return response;
 };
 
-export { getLocalStorage, setLocalStorage, fetchWithTimeout };
+const updateLinkWithConfiguredLangRoot = (link) => {
+  const langRoot = getMetadata('lang-root');
```

Comment:
Curious: Do you foresee other milo consumers using `lang-root` ?

Reply:
@chrischrischris Maybe? 
Probably not many unless their root content is different from the root (like the Blog). That's the main reason for this. It basically allows for a custom root.

Reply:
Would `customRoot`/`cusom-root` make better sense?  `lang-root` is my fault :)

Reply:
I'd keep it `lang-root` as it points to the language root of the site. For Blog, the language root is `/en`, not `/` which is the default language root for a Franklin/Milo based site.

Reply:
addressed in https://github.com/adobecom/milo/pull/1101/commits/e2149877f76d990b95a22a56b442badd24f5c7b6